mmv1/third_party/terraform/tests/resource_sql_database_instance_test.go

```diff
@@ -1036,6 +1036,34 @@ func TestAccSqlDatabaseInstance_cloneWithDatabaseNames(t *testing.T) {
 	})
 }
 
+func TestAccSqlDatabaseInstance_cloneWithPreferredZone(t *testing.T) {
+	// Sqladmin client
+	acctest.SkipIfVcr(t)
```

Comment:
What is the reason to skip this test in Vcr?

Reply:
Have you run this test locally to see if it passed?

Reply:
I run this test locally and got this error

`Invalid request: The preferred zone field isn't supported for the local-mediator-361721:tf-bootstrap-f0ruwbstwx instance..`

Do you know how to fix it? Thanks.

Reply:
This is the doc to run tests locally, in case you need it. https://googlecloudplatform.github.io/magic-modules/get-started/run-provider-tests/

Reply:
Dropped `preferred_zone` from this PR.